buildSrc/version.properties

```diff
@@ -55,3 +57,7 @@ bytebuddy         = 1.14.3
 
 # benchmark dependencies
 jmh               = 1.35
+
+jzlib = 1.1.3
+
+reactivestreams = 1.0.4
```

Comment:
https://github.com/opensearch-project/OpenSearch/blob/197086f515349c6e06321e56e4de9202f69b98a3/plugins/repository-azure/build.gradle#L58

We are using 1.0.3 in another plugin. Should we use the same version there to ensure going forward, we don't cause any conflicts?

Reply:
This seems reasonable, we should bring other plugins to the same versions so we don't face issues during upgrades too. I can raise another PR to consume package versions from here,